scripts/processMrw_bmc.pm

```diff
@@ -48,8 +50,34 @@ sub buildBMCAffinity
         }
         elsif ($type eq "DIMM")
         {
+            # The DIMM target that needs to be picked up has a new type
+            # as per the new HB changes to support DDR5. They will be
+            # unit-ddr* type and not lcard-dimm*. While parsing we get all
+            # of them, so skip the unit-ddr*, get the location code from
+            # the target with lcard type and set the location code for its
+            # child targets which are again unit-ddr*
+            my $target_type = $targetObj->getTargetType($target);
+            if (($target_type eq "unit-ddr") ||
+                ($target_type eq "unit-ddr4-jedec") ||
+                ($target_type eq "unit-ddr5-jedec"))
+               {
+                   next;
+               }
+
             my $location_code = getLocationCode($targetObj, $target);
+            # needed for Bonnell which has older DIMM type
             $targetObj->setAttribute($target, "LOCATION_CODE", $location_code);
+
+            foreach my $child (@{ $targetObj->getTargetChildren($target) })
```

Comment:
It's unclear to me, why are adding a LOCATION_CODE attribute to the unit targets. what is the use case?

Please point to the generated device tree for this.

Reply:
Refer /home/aravnair/sb_san5/repo/code/open-power/1060/forsdkbuild/patch
 You will see txt files which are attribute bumps with and without the changes plus the dts files.

Reply:
I have generated the device tree with this PR and noticed that the LOCATION_CODE attribute was added to the `dimm` target only

I thought we were adding one more child target as `unit_ddr` and in that adding the LOCATION_CODE attribute but, unit_ddr was treated as a dimm target so my comment is not valid.

Reply:
> I thought we were adding one more child target as `unit_ddr` and in that adding the LOCATION_CODE attribute but, unit_ddr was treated as a dimm target so my comment is not valid.

The "unit_ddr" from the MRW *is* the DIMM target on the platform.